src/cbmc/bmc.cpp

```diff
@@ -343,7 +343,7 @@ safety_checkert::resultt bmct::run(
     setup_unwind();
 
     // perform symbolic execution
-    symex(goto_functions);
+    symex(goto_functions, options.get_bool_option("atomic-nesting"));
```

Comment:
This option should be passed to the `symex_bmct` as other options, in the constructor of the `bmct`.

For the time being leave it like this, just squash a comment in this commit above this line with a FIXME explaining what's the better way of doing it.